gulpfile.cjs

```diff
@@ -421,6 +442,17 @@ gulp.task(
     //See https://github.com/CesiumGS/cesium/pull/3106#discussion_r42793558 for discussion.
     glslToJavaScript(false, "Build/minifyShaders.state");
 
+    // Remove prepare step from package.json to avoid redownloading Draco3d files
+    delete packageJson.scripts.prepare;
+    fs.writeFileSync(
+      "./package.noprepare.json",
```

Comment:
To keep the root directory clean let's put this file in the build folder:

`"Build/package.noprepare.json"`